module/VuFind/src/VuFind/Config/PathResolverFactory.php

```diff
@@ -82,14 +103,35 @@ public function __invoke(
         if (!empty($options)) {
             throw new \Exception('Unexpected options sent to factory.');
         }
-        $localDirs = defined('LOCAL_OVERRIDE_DIR')
-            && strlen(trim(LOCAL_OVERRIDE_DIR)) > 0
-                ? [
-                    [
-                        'directory' => LOCAL_OVERRIDE_DIR,
-                        'defaultConfigSubdir' => $this->defaultLocalConfigSubdir,
-                    ],
-                ] : [];
+        $localDirs = [];
+        $currentDir = defined('LOCAL_OVERRIDE_DIR')
+            && strlen(trim(LOCAL_OVERRIDE_DIR)) > 0 ?
```

Comment:
In our style, we prefer operators on the start of lines rather than the ends, so I'd recommend moving the `?` down to the start of the next line.

Reply:
fixed